iocore/net/SSLConfig.cc

```diff
@@ -343,6 +343,8 @@ SSLConfigParams::initialize()
   set_paths_helper(ssl_ocsp_response_path, nullptr, &ssl_ocsp_response_path_only, nullptr);
   ats_free(ssl_ocsp_response_path);
 
+  REC_ReadConfigInt32(ssl_handshake_timeout_in, "proxy.config.ssl.handshake_timeout_in");
```

Comment:
There should be a default values for the ssl handshake timeout that is non-zero, we don't use the default_inactivity_timeout.

Reply:
The documentation implies that when the configuration value is 0, the timeout is not implied.  Not sure if that means we should just not apply a timeout and leave it to whatever it was originally.  Or something else.  @sudheerv has another PR coming with a different approach which I think should be better.

https://docs.trafficserver.apache.org/en/9.0.x/admin-guide/files/records.config.en.html?highlight=handshake_timeout#proxy.config.ssl.handshake_timeout_in

Reply:
But in reality a 0 value is setting it to the default timeout.